controllers/serviceController_int_test.go

```diff
@@ -193,7 +193,7 @@ var _ = Describe("Creating ingress objects with the ingress generator", func() {
 			client, err := kubernetes.NewForConfig(ctrl.GetConfigOrDie())
 			Expect(err).NotTo(HaveOccurred())
 			deploymentWaiter := dogustart.NewDeploymentReadyChecker(client, "my-test-namespace")
-			waitOptions := dogustart.WaitOptions{Timeout: time.Minute * 30, TickRate: time.Millisecond * 200}
+			waitOptions := dogustart.WaitOptions{Timeout: 5, TickRate: 3}
```

Comment:
Is there a reason for this change? Lowering the timeout should not be a problem. However the tick rate should probably affect the whole time needed to run the unit test by at least 2,8 seconds.

Reply:
Reverted tickrate